druid/src/command.rs

```diff
@@ -384,6 +405,11 @@ impl Command {
         .default_to(Target::Global)
     }
 
+    /// Checks if this command must be used.
+    pub fn is_must_use(&self) -> bool {
```

Comment:
hmm, I find this name awkward, but the name I'd propose ("must_use") is used by the constructor. We could call the construct `new_must_use`, maybe, although that doesn't feel great either? We could also call this `must_be_used` which at least flows a bit better grammatically. I don't love any of these ideas... 😕

Reply:
I followed the second one because I expect `Selector::must_use("..")` to used much more than `Command::must_be_used`